### ISSUE TITLE
Request: have Android app too

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [sqlitebrowser/sqlitebrowser#1845]
This can be useful for use on-the-go.
Could use root if needed, to access private storage too.

**Comment by justinclift** [comment]
It's an idea (also iOS), but it'd probably need to be a separate application that we somehow integrate with.

That being said, we build upon [Qt](https://qt.io), which apparently has some mobile capabilities.

No idea how we'd cater to a touch screen interface though.  It'd likely take a bunch of effort. :wink:

**Comment by AndroidDeveloperLB** [comment]
I see. Could be very useful though, even if you start from something small that doesn't include all features we have today...

**Comment by justinclift** [comment]
What features would you start with? :smile:

**Comment by AndroidDeveloperLB** [comment]
You could divide it to 2 major features:
View, Edit.
Each is a big milestone.
I suggest to support rooted devices from the start, because this tool is usually used by developers, who need to check on their own app's DB files.

**Comment by justinclift** [comment]
Is it something you'd be interested in doing? :smile:

**Comment by AndroidDeveloperLB** [comment]
Yes, but :
1. I don't have this time. I already have 3 spare time apps:  https://play.google.com/store/apps/developer?id=AndroidDeveloperLB .
2. I need to know what to use of this repository, and how. I never looked at this code, and at this moment I'm pretty sure it's huge...

**Comment by justinclift** [comment]
No worries, it was a thought.  It's a decent idea, but it really comes down to people having the time + ongoing interest in developing it. :smile:

**Comment by AndroidDeveloperLB** [comment]
Sorry.
I was thinking that maybe it was in Java/Kotlin, when I first suggested it, so I thought that maybe it could be a bit easier.
There are some apps on the Play store for this, but most are either paid for or have full screen ads. 
Here it's free, without ads, and even open sourced...
I remember I saw an old app that was free without ads, but I can't find it anymore. Was probably removed.

**Comment by AndroidDeveloperLB** [comment]
Found the free, no-ads app that I told you about:
https://play.google.com/store/apps/d...ueller.dbadmin
Sadly it's not available anymore, but I can still download it.
Attached APK of it.
It's very easy to use, shows recently opened DB files, and all with minimal steps (all on main screen).

[SQLite Debugger - package oliver.ehrenmueller.dbadmin - versionCode 40.apk.zip](https://github.com/sqlitebrowser/sqlitebrowser/files/3067926/SQLite.Debugger.-.package.oliver.ehrenmueller.dbadmin.-.versionCode.40.apk.zip)

**Comment by justinclift** [comment]
That play.google.com link looks like a bad cut-n-paste?

**Comment by AndroidDeveloperLB** [comment]
@justinclift Right. Fixed. But again, it's not available for people who didn't download it.
Here's how it looks like:

![image](https://user-images.githubusercontent.com/5357526/55948048-5ede1180-5c58-11e9-912e-1bc08b674293.png)

**Comment by justinclift** [comment]
Cool. :smile:

**Comment by AndroidDeveloperLB** [comment]
I tried to contact the person who made it. Maybe it will become open sourced, so that you could start from something...
